src/Materials/Textures/babylon.renderTargetTexture.ts

```diff
@@ -20,7 +20,7 @@
         * Use this predicate to dynamically define the list of mesh you want to render.
         * If set, the renderList property will be overwritten.
         */
-        public renderListPredicate: (AbstractMesh: AbstractMesh) => boolean;
+        public renderListPredicate: (AbstractMesh) => boolean;

```

Comment:
It doesn't remove the type, just the parameter name since the name is redundant
ie
```typescript
let x: (number) => void;
is the same as 
let x: (param:number) => void;
```

nevertheless can restore if prefered